dnf-behave-tests/features/search.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Search command
+
+
+Background:
+  Given I use the repository "dnf-ci-fedora"
```

Comment:
Undefined step, correct one is: Given I use repository "dnf-ci-fedora"

Reply:
Thank you, fixed it.